tests/unit/test_main.py

```diff
@@ -3,12 +3,12 @@
 from __future__ import unicode_literals
 from mock import patch
 import httpretty
-from httpretty.core import HTTPrettyRequest
+from httpretty.core import HTTPrettyRequest, HTTPrettyRequestEmpty
 
 
 @patch('httpretty.httpretty')
 def test_last_request(original):
-    ("httpretty.last_request() should return httpretty.core.last_request")
+    """httpretty.last_request() should return httpretty.core.last_request"""
```

Comment:
I think I know why it was made into parenthesis although didn't work as "intended" -- I also dislike that nosetests uses tests docstring as what it outputs while running tests instead of the test name (thus making it easy to pick up and rerun the failed one instead of deducing which one was failing by name)